.circleci/config.yml

```diff
@@ -44,46 +31,37 @@ jobs:
     steps:
       - attach_workspace:
           at: /tmp/workspace
-      - run: cat /tmp/workspace/env-vars >> $BASH_ENV
+      - run: cat /tmp/workspace/env-vars >> ${BASH_ENV}
       - setup_remote_docker
       - run:
           name: Load archived Docker image
           command: docker load -i /tmp/workspace/image.tar
       - run:
           name: Publish Docker Image to Docker Hub as Sandbox
           command: |
-            echo "${DATESTAMP}"
-            echo "${DOCKERHUB_PASS}" | docker login -u "${DOCKERHUB_USERNAME}" --password-stdin
+            echo ${DOCKERHUB_PASS} | docker login -u ${DOCKERHUB_USERNAME} --password-stdin
             docker push ${IMAGE_NAME}:sandbox-${DATESTAMP}
-            docker run -e IMAGE_TO_SCAN=${IMAGE_NAME}:sandbox-${DATESTAMP} -e SYSDIG_SECURE_TOKEN=$SYSDIG_SECURE_TOKEN logdna/sysdigsecure
-  docker-approved:
-    executor: docker-publisher
-    steps:
-      - attach_workspace:
-          at: /tmp/workspace
-      - run: cat /tmp/workspace/env-vars >> $BASH_ENV
+            docker run -e IMAGE_TO_SCAN=${IMAGE_NAME}:sandbox-${DATESTAMP} -e SYSDIG_SECURE_TOKEN=${SYSDIG_SECURE_TOKEN} logdna/sysdigsecure
```

Comment:
I got curious about this logdna/sysdigsecure image, where is it built and released? I tried checking the LogDNA Github org but I did not find anything.

Reply:
@mindjiver, for now, I have decided to remove `test` and `build_windows` jobs because I'm gonna add later - this `PR` is mainly for cleaning code automating `Docker` image creation, scanning, pushing, etc.

for `logdna/sysdigsecure`, I have no idea, since it was done by @RYadav95 - maybe @gregswift can answer that